### PR TITLE
New Msys dependencies

### PR DESCRIPTION
**Pull request by hessenfarmer**
Finally the underlying issue of #6275 has been fixed in the MSYS repo. 
so we are finally able to build again without downgrading

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -330,7 +330,7 @@ jobs:
         pacman.exe --noconfirm -Syyuu --ignore ca-certificates
         pacman.exe --noconfirm -Syyuu --ignore ca-certificates
         bash.exe -c "./install-dependencies.sh msys${{ matrix.bits }} --noconfirm"
-        pacman.exe --noconfirm -U https://repo.msys2.org/mingw/mingw${{ matrix.bits }}/mingw-w64-${{ matrix.archname }}-SDL2_image-2.8.0-1-any.pkg.tar.zst
+        # pacman.exe --noconfirm -U https://repo.msys2.org/mingw/mingw${{ matrix.bits }}/mingw-w64-${{ matrix.archname }}-SDL2_image-2.8.0-1-any.pkg.tar.zst
```
Comment (tothxa) [inline .github/workflows/build.yaml:334]:
```suggestion
        # Downgrade incompatible packages to working version
        # pacman.exe --noconfirm -U https://repo.msys2.org/mingw/mingw${{ matrix.bits }}/mingw-w64-${{ matrix.archname }}-SDL2_image-2.8.0-1-any.pkg.tar.zst
```

**Comment by hessenfarmer** [comment]
works only for 64 bit. so lets wait if this will get fixed in the MSYS packages. (problem is already reported there

**Comment by hessenfarmer** [comment]
ok it seems we might need to deprecate 32 bit Windows builds at some time in the future (Maybe after 1.2) 
see https://github.com/msys2/MINGW-packages/issues/19510

**Review by tothxa — APPROVED** [review]
Thank you for all your work to sort this out.

It built successfully, and the changes are not unreasonable. I still find it unfortunate though that we now have such a long list of dependencies forced on us that we don't actually use ourselves.

**Review by matthiakl — APPROVED** [review]
> ok it seems we might need to deprecate 32 bit Windows builds at some time in the future (Maybe after 1.2)

+1

> I still find it unfortunate though that we now have such a long list of dependencies forced on us that we don't actually use ourselves.

At least the current setup allows for relatively easy building on Windows. If we compile the packages ourselves (like we do for MSVC) there is additional effort required to get a development environment running. Dynamic linking may pull less dependencies in, but causes other issues which are not alwalys found during build, but only during runtime.

@bunnybot update

**Comment by matthiakl** [comment]
@bunnybot update

**Comment by matthiakl** [comment]
@bunnybot merge

**Pull request closed without merging**